ansible/playbooks/start_morio.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Start Morio on the EC2 instances
+  hosts: all
+  become: yes
+  become_method: sudo
+  tasks:
+    - name: Start Morio service
+      shell: moriod start
```

Comment:
As a rule of thumb, we should only use shell in Ansible when nothing more suitable is available.

In this case, we can use the built-in systemd module: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/systemd_module.html

Reply:
I've rewrite it like this.

ansible.builtin.systemd:
name: moriod
state: started